src/main/java/modify_task_use_case/ModifyTaskInteractor.java

```diff
@@ -20,21 +20,30 @@ public ModifyTaskInteractor(
         this.dsGateway = dsGateway;
     }
 
+    /**
+     * @param inputData contains the dayIndex which contains the Task to be modified,
```

Comment:
Capitalize the first word after the parameter name. Applies to all javadocs.

Reply:
Noted. I shall address this in the next upcoming commit.